tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -338,6 +338,44 @@ async def test_get_balance(wallet_rpc_environment: WalletRpcTestEnvironment):
     await assert_get_balance(wallet_rpc_client, wallet_node, cat_wallet)
 
 
+@pytest.mark.asyncio
+async def test_get_farmed_amount(wallet_rpc_environment: WalletRpcTestEnvironment):
+    env = wallet_rpc_environment
+    wallet: Wallet = env.wallet_1.wallet
+    full_node_api: FullNodeSimulator = env.full_node.api
+    wallet_rpc_client = env.wallet_1.rpc_client
+    wallet_node: WalletNode = env.wallet_1.node
+    await full_node_api.farm_blocks_to_wallet(2, wallet)
+
+    result = await wallet_rpc_client.get_farmed_amount()
+
+    expected_result = {
+        "farmed_amount": 4000000000000,
+        "farmer_reward_amount": 500000000000,
+        "fee_amount": 0,
+        "last_height_farmed": 2,
+        "pool_reward_amount": 3500000000000,
+        "success": True,
+    }
+
+    assert result == expected_result
+
+    our_ph = await wallet.get_new_puzzlehash()
```

Comment:
Is it important to not have this be two test runs with two independent results?  Since the fee amount is reported separately I think the fee case could actually not test the amounts at all and not be codependent.

Reply:
Yea, makes some sense, so to clarify, the suggestion is an entirely second test

Reply:
I think so here, yeah.  At first I thought it could be parametrized, but I think it can be more isolated than that without drastic code duplication.  See what it looks like.